extensions/wikia/Email/templates/userNameLayout.mustache

```diff
@@ -0,0 +1,20 @@
+{{> greeting}}
+<table>
+	<tr>
+		<td class="border">
+			<div class="avatar-wrapper">
+				<img class="avatar" src="{{editorAvatarURL}}">
+			</div>
+		</td>
+		<td class="widest border">
+			<div class="message">
+				<p>
+					{{userNameMessage}}
+				</p>
+			</div>
+		</td>
+	</tr>
+	{{# hasContentFooterMessages}}
```

Comment:
Is this template used by a controller which does't have contentFooterMessages? If not, can we get rid of this conditional?